meta/main.yml

```diff
@@ -14,6 +14,7 @@ galaxy_info:
       versions:
         - "8"
         - "9"
+        - "10"
```

Comment:
```suggestion
        - "10"
```
Ansible lint does not recognize "10" as a valid version.  In fact, Ansible is deprecating this version mechanism altogether.  `galaxy_tags` is now used for versions going forward.

Reply:
Ah, ok. Good to know. I dropped the commit changing this and kept only the one fixing the repositories.